### ISSUE TITLE
.Net4.0 Support

### ISSUE THREAD

**Issue by wahmedswl** [benaadams/Ben.Demystifier#26]
Hi,
   It would be great if this package target .net4.0 as it is the last Long Term release and heavily deployed. Also, it would be great if Demystifier can work in StackTrace string as that would really much increase its usability even in CI Servers to post-process data. 

Thanks for this great package

**Comment by benaadams** [comment]
>  target .net4.0 as it is the last Long Term release

Are you thinking 4.5.2?

Support for .NET 4.0, 4.5, 4.5.1 ended last year on January 12, 2016

**Comment by wahmedswl** [comment]
yup, you are right but we target .net4.0 and this was something missing for long time. We also thinking to use Sentry which also demangle names but using Regex and that approach works and can be used everywhere. Having something from .Net team is awesome and might be this stuff got integrated in .Net itself if it proves its worth.

Thanks for your quick response

**Comment by kiquenet** [comment]
.NET Framework 4.7.2 support ?

**Comment by bruno-garcia** [comment]
@kiquenet it's already supported

![image](https://user-images.githubusercontent.com/1633368/103490253-986d8d80-4de8-11eb-84ad-128dd9c9113b.png)


From .NET Framework 4.5

**Comment by kiquenet** [comment]
Applies to NET 4.8, NET Core 3.2 and NET 5 ?

**Comment by benaadams** [comment]
Yes and the CI runs the tests for `net46`+, `netcoreapp2.1`, `netcoreapp3.1` and `net5.0`

![image](https://user-images.githubusercontent.com/1142958/104211554-1b679700-542c-11eb-85de-0f4ee85e027a.png)
